kernel/config.txt

```diff
@@ -82,3 +214,31 @@ script. This fully removes the perl requirement.
 |   TIP: All links like this one are also available via 'kiss help'!           |
 |                                                                              |
 +------------------------------------------------------------------------------+
+
+
+[6.0] Making an initramfs (optional)
+________________________________________________________________________________
+
+An initramfs is an interesting and messy beast to handle. Put simply, it is a
+compressed cpio archive, which contains a very minimal root filesystem with some
+kernel modules and firmware, whose only job is to prepare and mount the actual
+root filesystem.
+
+Why? It is useful for several objectives. A "generic" kernel configuration
+becomes too large to be practical, since a kernel needs to build-in all the
+drivers necessary for running core hardware and mounting the actual root disk
+and filesystem. It is also very useful if you want to encrypt your root
+partition using FDE.
+
+Again, an initramfs is not mandatory in any way. If you build all the required
+drivers into a kernel (since modules are themselves stored in, well, disks and
+filesystems) and don't have any special uses for it, you will get a perfectly
+booting systen without an initramfs. There are even workarounds to make FDE
+decryption work without an initramfs.
+
+Generating initramfs files is a difficult task. Many distros have their own
+tools for doing it, but they are largely non-portable, complicated and unwieldy.
+
+$/illiliti/tinyramfs is a portable (though built for KISS) initramfs generator.
+It is still in an early development stage and has yet to support several
+features, but is largely useable and pretty great to tinker with for your uses.
```

Comment:
I feel that this belongs in its own page (one dedicated to using an initramfs). :+1: